doc/install-macos.md

```diff
@@ -0,0 +1,78 @@
+# macOS Installation
+
+To install the fluxon-mhd modeling framework, there are three main requirements - a perl installation with libraries, the perl data language toolset, and finally compiling the fluxon-mhd software itself. Some of the background perl installation requires root / admin access, but beyond that packages can be installed locally for ease of editing and recompilation.
+
+## Perl installation
+
+First, install the homebrew package, which will require sudo access.
+```shell
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Comment:
this command has instructions that it gives in the terminal that the user should be reminded to do (or, we automate it)

Reply:
Oh, I suppose that line 35 does this, sort of

Reply:
I'm wondering if leaving this out is breaking things for me. 
Maybe the command needs to be done in an all-admin setup?